pkg/ops/iso.go

```diff
@@ -31,33 +32,49 @@ func GenISO(name, src, dst string, i schema.ISO) func(ctx context.Context) error
 			overlay = i.DataPath
 		}
 
+		// We are assuming StepCopyCloudConfig has already run, putting it the config in "dst"
 		err = copy.Copy(filepath.Join(dst, "config.yaml"), filepath.Join(overlay, "config.yaml"))
 		if err != nil {
 			return err
 		}
 
-		internal.Log.Logger.Info().Msgf("Generating iso '%s' from '%s' to '%s'", name, src, dst)
+		internal.Log.Logger.Info().Msgf("Generating iso '%s' from '%s' to '%s'", i.Name, src, dst)
 		cfg := enkiconfig.NewBuildConfig(
 			enkiconfig.WithLogger(sdkTypes.NewKairosLogger("enki", "debug", false)),
 		)
-		cfg.Name = name
+		cfg.Name = i.Name
 		cfg.OutDir = dst
-		// Live grub artifacts:
-		// https://github.com/kairos-io/osbuilder/blob/95509370f6a87229879f1a381afa5d47225ce12d/tools-image/Dockerfile#L29-L30
-		// but /efi is not needed because we handle it here:
-		// https://github.com/kairos-io/enki/blob/6b92cbae96e92a1e36dfae2d5fdb5f3fb79bf99d/pkg/action/build-iso.go#L256
-		// https://github.com/kairos-io/enki/blob/6b92cbae96e92a1e36dfae2d5fdb5f3fb79bf99d/pkg/action/build-iso.go#L325
+		cfg.Date = i.IncludeDate
+		if i.Arch != "" {
+			cfg.Arch = i.Arch
+		}
+		isoLabel := enkiconstants.ISOLabel
+		if i.Label != "" {
+			isoLabel = i.Label
+		}
 		spec := &enkitypes.LiveISO{
 			RootFS:             []*v1.ImageSource{v1.NewDirSrc(src)},
 			Image:              []*v1.ImageSource{v1.NewDirSrc("/grub2"), v1.NewDirSrc(overlay)},
-			Label:              "COS_LIVE",
+			Label:              isoLabel,
```

Comment:
as said above, this is non-negotiable :D

Reply:
yup, changed

Reply:
Actually, this one is needed because it's used here: https://github.com/kairos-io/enki/blob/60311c6cc4dff0ed150e79ad57a4e37b67ddf25b/pkg/action/build-iso.go#L409 and I don't see it being set to the default `enkiconstants.ISOLabel` anywhere else. I just removed the blog that allows the user to overwrite it (lines 52-54 above)